cli/integration-test/integration/completion_test.clj

```diff
@@ -0,0 +1,45 @@
+(ns integration.completion-test
+  (:require
+   [clojure.test :refer [deftest testing]]
+   [integration.fixture :as fixture]
+   [integration.helper :as h]
+   [integration.lsp :as lsp]))
+
+(lsp/clean-after-test)
+
+(deftest completion
+  (lsp/start-process!)
+  (lsp/request! (fixture/initialize-request
+                 {:initializationOptions {:lint-project-files-after-startup? false}
+                  :capabilities
+                  {:textDocument
+                   {:completion
+                    {:contextSupport true
+                     :completionItemKind {:valueSet (vec (range 1 26))}
```

Comment:
Why we need this line? I don't see emacs sending it

Reply:
This is one of the capabilities Calva sends. I wanted to stay kind of close to the actual usage, so that the test works if lsp ever starts checking that capability. I'm happy to remove it though.